null_statsd.gemspec

```diff
@@ -1,25 +1,26 @@
 # coding: utf-8
-lib = File.expand_path('../lib', __FILE__)
+lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'null_statsd/version'
+require "null_statsd/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = "null_statsd"
-  spec.version       = NullStatsd::VERSION
-  spec.authors       = ["Imad Mouaddine"]
-  spec.email         = ["imouaddine@usertesting.com"]
+  spec.name = "null_statsd"
```

Comment:
Awww, you made the `=` alignments all ugly 😜 😢

Reply:
I fixed it